spec/models/guest_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+RSpec.describe Guest, type: :model do
+  describe "relationships" do
+    it {should have_many(:room_guests)}
```

Comment:
Nice work with this model test. We'd also want to see a model spec for RoomGuest, and the additional relationships in the Room spec. It's a good idea to get these model specs set up before you even head into the first user story.